### DISCUSSION TITLE
Health check failing because of Istio gateway

### DISCUSSION THREAD

**Discussion by GBRGUP2** [istio/istio#49952] [Q&A]
### Is this the right place to submit this?

- [X] This is not a security vulnerability or a crashing bug
- [X] This is not a question about how to use Istio

### Bug Description

Hi , need urgent help as this has started impacting our business. Health checks of all the apps are intermitently failing in one of the clusters because of istio ingress gateway but other cluster which is having same configuration is having no problem. We are having this problem since last week and we are stuck in deadlock. Any help will be appreciated

### Version

```Text
client version: 1.20.3
istiod version: 1.19.3
istiod version: 1.19.3
pilot version: 1.17.3
pilot version: 1.17.3
data plane version: 1.17.3 (334 proxies), 1.19.3 (11 proxies)
```

**Comment by howardjohn** [comment]
Without more information about the setup like logs, configurations, etc this is going to be impossible for anyone to help with.
